packages/verification-vue-component/src/style/components/verification/_dropbox.scss

```diff
@@ -52,4 +48,41 @@
             }
         }
     }
+
+    &.dragover {
+        position: absolute;
```

Comment:
We should make sure that this absolute position stays inside this component. So I would add position: relative; and overflow: hidden; to .certifaction-verification, of course we then need to make sure, that it uses all the space it has, probably add width: 100%; height: 100%; there, and in the verification app we would also make sure that this component can stretch to the whole page.